src/applications/appeals/10182/pages/contestableIssues.js

```diff
@@ -19,6 +20,7 @@ const contestableIssues = {
       'ui:title': ' ',
       'ui:field': 'StringField',
       'ui:widget': ContestableIssuesWidget,
+      'ui:required': formData => !hasSomeSelected(formData),
```

Comment:
is this confirming that there's at least one issue selected since selecting an issue is required to proceed?

Reply:
So if this was set to `'ui:required': () => true` it would expect an API loaded issue to be included. And since we expect at least one issue from `contestableIssues` or `additionalIssues` to be checked. I might need to do some additional testing to make sure this is working as expected - or remove it and add this work in a separate PR!

Reply:
I'm going to comment out these lines until I do some more checking.